pbpy/pbtools.py

```diff
@@ -198,36 +197,32 @@ def resolve_conflicts_and_pull():
     # Make sure upstream is tracked correctly
     pbgit.set_tracking_information(pbgit.get_current_branch_name())
 
-    pblog.info("Trying to stash the local work...")
-    output = subprocess.getoutput(["git", "stash"])
-    pblog.info(str(output))
-
     pblog.info(
         "Trying to rebase workspace with latest changes on the repository...")
-    output = subprocess.getoutput(["git", "pull", "--rebase"])
+    output = subprocess.getoutput("git pull --rebase --autostash")
     pblog.info(str(output))
 
     lower_case_output = str(output).lower()
 
     if "failed to merge in the changes" in lower_case_output or "could not apply" in lower_case_output:
-        pblog.error("Aborting the rebase. Changes on one of your commits will be overridden by incoming changes. Request help on #tech-support to resolve conflicts, and  please do not run StartProject.bat until issue is solved.")
+        pblog.error(
+            "Aborting the rebase. Changes on one of your commits will be overridden by incoming changes. Request help in #tech-support to resolve conflicts, and please do not run StartProject.bat until the issue is resolved.")
         pbgit.abort_rebase()
-        pbgit.stash_pop()
         error_state(True)
     elif "fast-forwarded" in lower_case_output:
-        pbgit.stash_pop()
-        pblog.info("Success, rebased on latest changes without any conflict")
+        pblog.info("Success, rebased on latest changes without any conflicts")
     elif "is up to date" in lower_case_output:
-        pbgit.stash_pop()
-        pblog.info("Success, rebased on latest changes without any conflict")
-    elif "rewinding head" in lower_case_output and not("error" in lower_case_output or "conflict" in lower_case_output):
-        pbgit.stash_pop()
-        pblog.info("Success, rebased on latest changes without any conflict")
+        pblog.info("Success, rebased on latest changes without any conflicts")
+    elif "rewinding head" in lower_case_output and not (
+            "error" in lower_case_output or "conflict" in lower_case_output):
+        pblog.info("Success, rebased on latest changes without any conflicts")
     elif "successfully rebased and updated" in lower_case_output:
-        pbgit.stash_pop()
-        pblog.info("Success, rebased on latest changes without any conflict")
+        pblog.info("Success, rebased on latest changes without any conflicts")
     else:
-        pblog.error("Aborting the rebase, an unknown error occured. Request help on #tech-support to resolve conflicts, and please do not run StartProject.bat until issue is solved.")
+        pblog.error(
+            "Aborting the rebase because of an unknown error. Request help in #tech-support to resolve it, and please do not run StartProject.bat until the issue is resolved.")
         pbgit.abort_rebase()
-        pbgit.stash_pop()
         error_state(True)
+
+    pblog.info("Cleaning up unused repository assets...")
+    output = subprocess.getoutput("git-lfs prune -c")
```

Comment:
Running this on each launch is not a good idea. It adds about 20-25 seconds to editor launch time.

Reply:
How about spawning a separate background process which allows PBSync to progress?

Reply:
If you have time to implement that, why not?

Reply:
Any update on this?